### PR TITLE
Move filters to top of page

### PR DESCRIPTION
**Pull request by cmc333333**
This gets us a bit closer to the mockup, but still needs lots of style love. It moves filters from the sidebar to the top of the page and makes policy filters visible.

screen shot 2017-05-09 at 11 46 30 am

### CODE REVIEW

**Comment by theresaanna** [comment]
Does this look better? @micahtaylor @cmc333333 
screen shot 2017-05-09 at 1 11 29 pm

**Comment by micahtaylor** [comment]
Just a bit more spacing on the left to match the spacing on the right would be ideal. Looks great!

**Comment by theresaanna** [comment]
@micahtaylor idek how you saw that difference, but it should be fixed!

File: ui/assets/img/remove-btn.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
+    <title>Remove btn</title>
+    <desc>Created with Sketch.</desc>
```
Comment (micahtaylor) [inline ui/assets/img/remove-btn.svg]: Bah! This is bad on me. I need to figure out why Sketch is not exporting compressed svg for me. It's fine for now. I'll create a separate issue to deal with it later.

**Comment by cmc333333** [comment]
Nice teamwork all!

**Pull request merged**